src/allotropy/parsers/utils/pandas.py

```diff
@@ -121,11 +121,13 @@ def split_header_and_data(
 def split_dataframe(
     df: pd.DataFrame,
     should_split_on_row: Callable[[pd.Series[Any]], bool],
+    include_split_row: bool = False,  # noqa: FBT001, FBT002
```

Comment:
To get rid of these qa warnings, put a * in arguments (this forces include_split_row to be kwarg only)
```suggestion
    *, include_split_row: bool = False,
```